PurchasesTests/Networking/BackendTests.swift

```diff
@@ -1133,19 +1140,19 @@ class BackendTests: XCTestCase {
         let path = "/subscribers/" + userID + "/offerings"
         httpClient.mock(requestPath: path, response: response)
 
-        var receivedError : NSError?
-        var receivedUnderlyingError : NSError?
-        backend?.getOfferings(appUserID: userID, completion: { (offeringsData, error) in
+        var receivedError: NSError?
+        var maybeReceivedUnderlyingError: NSError?
```

Comment:
I don't know if I understand what you mean 🤔 AFAIK, there is convection about optional properties in the [SwiftStyleGuide](https://github.com/RevenueCat/purchases-ios/blob/79449bf53fc8113c0c479f0841a17e4e940d190e/SwiftStyleGuide.swift#L57) file, no?

Reply:
Yeah so ignore me 😛 I just think it's weird and nobody else uses that convention 🙊

Reply:
[not for this PR] happy to re-evaluate the convention. 
We got it originally from a WWDC video, but I _think_ that video may have had obj-c code, where the _maybe_ is a useful visual reminder because nullability isn't enforced the way that it is enforced for swift. 

After migrating the codebase to swift, I don't feel like it adds much, other than being mildly helpful for code inside completion blocks. But the compiler forces handling of nil values anyway.